packages/app-desktop/gui/ClipperConfigScreen.jsx

```diff
@@ -39,6 +40,16 @@ class ClipperConfigScreenComponent extends React.Component {
 		alert(_('Token has been copied to the clipboard!'));
 	}
 
+	renewToken_click() {
+		if (confirm(_('Are you sure you want to renew the authorisation token?'))) {
+			void EncryptionService.instance()
+				.randomHexString(64)
```

Comment:
Could you create a new function under EncryptionService called something "generateApiToken" (which would in turn call randomHexString) and call this from here? Then also please update the initial call where the API token is set. This is just to avoid duplicate code.

Reply:
Sure, I can do that.

I actually thought about doing it, but then I saw that it is just one function call, so I wasn't sure anymore. But yeah, I will do that when I get up.

Reply:
Yes small thing, but if the token can be generated in two different place I think it makes sense to have one central function to do the job.

Reply:
Done.